pandas/io/gbq.py

```diff
@@ -338,7 +338,10 @@ def read_gbq(query, project_id = None, index_col=None, col_order=None, reauth=Fa
         page = pages.pop()
         dataframe_list.append(_parse_data(schema, page))
 
-    final_df = concat(dataframe_list, ignore_index = True)
+    if len(dataframe_list)>0:
+        final_df = concat(dataframe_list, ignore_index = True)
+    else:
+        final_df =  _parse_data(schema,[])
```

Comment:
This makes it necessary. Since we are constructing blank dataframe, it should pick the schema from query response.